editor/src/components/canvas/ui-jsx-canvas.test-utils.tsx

```diff
@@ -49,6 +49,8 @@ import { NO_OP } from '../../core/shared/utils'
 import { directory } from '../../core/model/project-file-utils'
 import { contentsToTree } from '../assets'
 import { MapLike } from 'typescript'
+import { generateCodeResultCache } from '../custom-code/code-file'
```

Comment:
Looks like this import wasn't needed in the end

Reply:
```suggestion
```